docs/rules/import-style.md

```diff
@@ -0,0 +1,87 @@
+# Enforce specific import styles per module
+
+Sometimes a module contains unrelated functions like `util`, thus it is a good practice to enforce destructuring or named imports here, other times in modules like `path` it is good to do default import as they have similar functions, all likely to be utilised.
+
+This rule defines 4 import types:
+* `unassigned` - `import 'foo'` or `require('foo')`
```

Comment:
Isn't this called `empty import`?

Reply:
The name is inspired by [no-unassigned-import](https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/no-unassigned-import.md) rule